tesler-doc-ui/package.json

```diff
@@ -5,7 +5,7 @@
   "homepage": "http://idocs.tesler.io/",
   "license": "Apache-2.0",
   "scripts": {
-    "build": "node --max_old_space_size=1024 node_modules/.bin/webpack --mode=production",
+    "build": "webpack --mode=production",
```

Comment:
I suspect it'll end up with out of memory during deployment on http://idocs.tesler.io/.
There are possible solutions:
1) Configure swap usage on documentation server to use disk space during the build stage
2) Separate build scripts as `build` and `build:limit` and change the deployment pipeline to use the latter
3) Modify build script to work both on Windows and *nix systems

Reply:
Seems to be concrete build server specific problem. 
Will it be ok to just set evn variable on this server?
E.g. change http://idocs.tesler.io:8443/job/tesler_doc_deploy_github/configure 
from:
```
withMaven(
                        mavenSettingsConfig: '3895a236-7a74-4276-8018-0fcd24d39374',
                        options: [
                                artifactsPublisher(disabled: true)
                        ]
                ) {
                    sh("mvn --projects ${getBaseProject()} -PIntegrationTests,UI,tesler_db help:effective-pom -Doutput=${getResultingPom()}")
                    doUnDeploy()
                    sh("mvn clean verify --projects tesler-doc-it/tesler-doc-it-db -am -PIntegrationTests,Tests,UI,tesler_db -Dliquibase.dropfirst=${isDropAll()}")
                    doDeploy()
                }
```
to:
```
withMaven(
                        mavenSettingsConfig: '3895a236-7a74-4276-8018-0fcd24d39374',
                        options: [
                                artifactsPublisher(disabled: true)
                        ]
                ) {
                    sh("set NODE_OPTIONS=--max_old_space_size=1024")
                    sh("mvn --projects ${getBaseProject()} -PIntegrationTests,UI,tesler_db help:effective-pom -Doutput=${getResultingPom()}")
                    doUnDeploy()
                    sh("mvn clean verify --projects tesler-doc-it/tesler-doc-it-db -am -PIntegrationTests,Tests,UI,tesler_db -Dliquibase.dropfirst=${isDropAll()}")
                    doDeploy()
                }
```

Reply:
Solved by `Limit` profile that will be used in pipeline